### ISSUE TITLE
Fallout 4 Mods Wont load

### ISSUE THREAD

**Issue by gabeorlanski** [TanninOne/modorganizer#825]
The mods will not load when launching the game no matter what I do

**Comment by PwDBiX** [comment]
I got the same problem like @gabeorlanski  and I can not figure out what is causing it. The plugin.txt file is looking as it should with all my installed mods showing up on the list but still Fallout 4 does not recognize that they are installed.
After reinstalling MO and testing it again with various mods (Script/texture-mods) I installed NMM and fallout 4 recognized my mods with zero issues. Unfortunately the last time I used MO with Fallout 4 it was version Alpha 4 I believe so I am not able to tell if it came with v2.0.7.

**Comment by RPINerd** [comment]
Do either of you have bethesda.net mods installed as well? I haven't tried it out with the 2.0.7, but with 2.0.6 I experienced some random issues with the "built in" mod handler messing with Mod Organizer installs. At the same time NMM would work fine which I presume is because it actually physically installs the mod into the Fallout 4 directory? But that's more just speculation on my part.

**Comment by WhiteZeroX** [comment]
Sounds similar to my issue #697

**Comment by PwDBiX** [comment]
@RPINerd : No, I am not using the "built in" mod handler to avoid problems with Mod Managers like NMM and MO, but I guess you are right about NMM and why its working while MO struggles. 

@WhiteZeroX : Yep, read through your posts, its the same issue. Tried to delete Fallout4.esm from plugins.txt like you did, same outcome.
